0.X/Dockerfile

```diff
@@ -13,6 +13,8 @@ ENV HASHICORP_RELEASES=https://releases.hashicorp.com
 RUN addgroup consul && \
     adduser -S -G consul consul
 
+RUN apk update && apk add ca-certificates  && update-ca-certificates  && apk --no-cache add openssl wget
```

Comment:
Compound RUN commands like this are usually split across lines, see the addgroup above.